plugins/generic/usageEvent/UsageEventPlugin.inc.php

```diff
@@ -286,6 +286,18 @@ function _buildUsageEvent($hookName, $args) {
 			$request, null, $canonicalUrlPage, $canonicalUrlOp, $canonicalUrlParams
 		);
 
+		// Make sure we log the server name and not aliases.
+		$configBaseUrl = Config::getVar('general', 'base_url');
+		$requestBaseUrl = $request->getBaseUrl();
+		if ($requestBaseUrl !== $configBaseUrl) {
+			// Make sure it's not an url override (no alias on that case).
+			if (!in_array($requestBaseUrl, Config::getContextBaseUrls()) && 
+					$requestBaseUrl !== Config::getVar('general', 'base_url[index]')) {
```

Comment:
The Apache virtualhost uses mod_write to point a set of hosts to a particular journal.

For example

```
<VirtualHost *:80>
    ServerAdmin "webmaster@DOMAINNAME"
    ServerName DOMAINNAME
    #ServerAlias www.DOMAINNAME
    MIMEMagicFile /dev/null
    CustomLog logs/DOMAINNAME.access_log "%h %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-agent}i\""
    ErrorLog logs/DOMAINNAME.error_log

    # include our standard se exchange vhost file
    Include conf.d/se_vhost/se_standard_vhost/se_standard_vhost.conf

    RewriteEngine on
    RewriteRule ^/(index.php)?/?$ /ojs/index.php/JOURNALNAME/index [L]

    RewriteRule ^/JOURNALNAME/(.*)$ /$1 [R,L]

    RewriteCond %{REQUEST_URI} !/index.php/
    RewriteCond %{DOCUMENT_ROOT}ojs%{REQUEST_URI} !-d
    RewriteCond %{DOCUMENT_ROOT}ojs%{REQUEST_URI} !-f
    RewriteCond %{REQUEST_FILENAME} !-d
    RewriteCond %{REQUEST_FILENAME} !-f
    RewriteRule ^(.*)$ /ojs/index.php/JOURNALNAME/$1 [QSA,L]

    RewriteCond %{DOCUMENT_ROOT}ojs%{REQUEST_URI} -f
    RewriteRule ^(.*)$ /ojs/$1 [QSA,L]

</VirtualHost>
```

So, HTTP_HOST could end up as ANY of the ServerName or ServerAliases, and the REQUEST_URI will mask the /ojs/index.php/JOURNALNAME.

This is why I think `Config::getVar('general', 'base_url['.$journalKey.']')` is required.